<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindVisitDialog</class>
 <widget class="QDialog" name="FindVisitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пошук відвідувань</string>
  </property>
  <widget class="QGroupBox" name="mainGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Пошук відвідувань</string>
   </property>
   <widget class="QLabel" name="facultyLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Факультет:</string>
    </property>
   </widget>
   <widget class="QLabel" name="groupLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Група:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="dateGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Дата</string>
    </property>
    <widget class="QDateEdit" name="beginDateEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="endDateEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>з:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>по:</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="facultyEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="groupEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="findButton">
       <property name="text">
        <string>Пошук</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevertButton">
       <property name="text">
        <string>Очистити</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Закрити</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="diagnosisLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Хвороба:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="diagnosisEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
